client/app/bundles/course/video/submission/actions/video.js

```diff
@@ -144,36 +144,103 @@ export function seekEnd() {
 /**
  * Creates an action to remove events from the state store.
  * @param sequenceNums The event sequence numbers to remove
+ * @param sessionConfirmed If this event is in response to the final server request before close
```

Comment:
Is a session "confirmation" a technical term? If not, would "sessionEnded" or "sessionClosed" be clearer?

Reply:
I think typically it's called "committed", but I think "sessionClosed" will work.

Reply:
Do you have a link where I can read up more on it? From a quick search, it seems to be a platform specific term.

Reply:
"Commit" is a distributed database term actually. I suggested it because I started the approach based on https://en.wikipedia.org/wiki/Two-phase_commit_protocol, but I don't think it serves our purposes here anymore.

Reply:
It is common to say that a transaction is committed, but I'm not sure in which contexts sessions said to be committed. In any case, we can go with "sessionClosed".